regparser/tree/xml_parser/reg_text.py

```diff
@@ -171,15 +180,29 @@ def build_from_section(reg_part, section_xml):
         if ch.tag == 'STARS':
             nodes.append(Node(label=[mtypes.STARS_TAG]))
         elif not markers_list:
-            section_texts.append((text, tagged_text))
+            # is this a bunch of definitions that don't have numbers next to them?
+            if subject_text.find('Definitions.') > -1:
+                if text.find('means') > -1:
+                    def_marker = text.split('means')[0].strip().split()
+                    def_marker = ''.join([word[0].upper() + word[1:] for word in def_marker])
+                else:
+                    def_marker = 'def{0}'.format(i)
+                n = Node(text, label=[def_marker], source_xml=ch)
+                n.tagged_text = tagged_text
+                n.defn_no_number = True
+                nodes[-1].children.append(n)
+            else:
+                section_texts.append((text, tagged_text))
```

Comment:
Also, you'll definitely want some tests around this. Regression bugs pop up a lot with the parser; best to nip them in the bud with uni ttests rather than try to debug reg Z :)